packages/dashmate/src/status/scopes/platform.js

```diff
@@ -11,54 +11,131 @@ const providers = require('../providers');
  */
 function getPlatformScopeFactory(dockerCompose,
```

Comment:
if service is not running `ServiceIsNotRunningError: Service drive_tenderdash for testnet is not running. Please run the service first.`